README.md

```diff
@@ -1,40 +1,44 @@
 <div align="center">
 
-  <h1><code>near-bindgen</code></h1>
+  <h1><code>near-sdk</code></h1>
 
   <p>
     <strong>Rust library for writing NEAR smart contracts.</strong>
   </p>
+  <p>
+    Previously known as <code>near-bindgen</code>.
+  </p>
+
 
   <p>
-    <a href="https://crates.io/crates/near-bindgen"><img src="https://img.shields.io/crates/v/near-bindgen.svg?style=flat-square" alt="Crates.io version" /></a>
-    <a href="https://crates.io/crates/near-bindgen"><img src="https://img.shields.io/crates/d/near-bindgen.svg?style=flat-square" alt="Download" /></a>
+    <a href="https://crates.io/crates/near-sdk"><img src="https://img.shields.io/crates/v/near-sdk.svg?style=flat-square" alt="Crates.io version" /></a>
+    <a href="https://crates.io/crates/near-sdk"><img src="https://img.shields.io/crates/d/near-sdk.svg?style=flat-square" alt="Download" /></a>
     <a href="https://discord.gg/gBtUFKR"><img src="https://img.shields.io/discord/490367152054992913.svg" alt="Join the community on Discord" /></a>
-    <a href="https://travis-ci.com/nearprotocol/near-bindgen"><img src="https://travis-ci.com/nearprotocol/near-bindgen.svg?branch=master" alt="Travis Build" /></a>
+    <a href="https://travis-ci.com/nearprotocol/near-sdk"><img src="https://travis-ci.com/nearprotocol/near-sdk.svg?branch=master" alt="Travis Build" /></a>
   </p>
 
    <h3>
-      <a href="https://github.com/nearprotocol/near-bindgen#features">Features</a>
+      <a href="https://github.com/nearprotocol/near-sdk#features">Features</a>
```

Comment:
Should we just make these links relative? `#features` vs `https://github.com/nearprotocol/near-sdk#features`?